internal/k8s/appprotect/app_protect_resources.go

```diff
@@ -131,6 +131,15 @@ func ParseResourceReferenceAnnotation(ns, antn string) string {
 	return antn
 }
 
+// ParseResourceReferenceAnnotationList returns a slice of ns/names strings
+func ParseResourceReferenceAnnotationList(ns, antns string) []string {
```

Comment:
Optional: Maybe this should be just called `annotations`? For a moment I understand `antns` to mean `annotationNamespace`
```suggestion
func ParseResourceReferenceAnnotationList(ns, annotations string) []string {
```

```suggestion
func ParseResourceReferenceAnnotationList(ns, antnRefs string) []string {
```